cypress/e2e/login_management/forgotPasswordDialog.feature

```diff
@@ -1,29 +1,33 @@
-@unstable_test
-Feature: Login Management - Visibility of dBildungscloud Password Reset dialog
+@regression_test
+@stable_test
+Feature: Login Management - Forgot Password Dialog Visibility Checks
 
     As a user (teacher or student or admin), I want to see password recovery dialog box visible to me so that I can recover my password in case needed.
 
-    @unstable_test
-    Scenario Outline: User sees password recovery dialog
+
+    Scenario: All elements in I see the forgot password dialog is visible
```

Comment:
Shouldn't it be "All elements of the Passwort Reset dialog are available"

Reply:
done!